tests/test_no_imports.rs

```diff
@@ -22,6 +22,21 @@ fn basic_module(_py: pyo3::Python<'_>, m: &pyo3::types::PyModule) -> pyo3::PyRes
     Ok(())
 }
 
+#[pyo3::pymodule]
+fn basic_module_bound(
+    _py: pyo3::Python<'_>,
```

Comment:
Was this meant to take just a single argument?

```suggestion
```

Reply:
Good catch. Fixed.